README.md

```diff
@@ -7,78 +7,6 @@
 The CanDIG v2 project is a collection of heterogeneous services designed to work together to facilitate end to end
 dataflow for genomic data.
 
-```plaintext
```

Comment:
Good point. I dumped the figure into that confluence doc. Agreed that it would be good to have an updated document in this repo, but I think that is a separate ticket.